context/app/static/js/components/detailPage/Protocol/Protocol.jsx

```diff
@@ -22,6 +24,16 @@ function ProtocolLink({ title, resolverHostnameAndDOI }) {
 }
 
 function Protocol({ protocol_url }) {
+  const {
+    entity: { entity_type },
+  } = useFlaskDataContext();
+
+  const tooltipTexts = {
```

Comment:
@tsliaw

Reply:
@NickAkhmetov I used the word `provided` since I was thinking of how the data submitter "provides" the protocols to protocols.io, but I see the confusion now with how I worded it in this tooltip. I'm not sure about the word `hosted` either. Does `Protocols provided for the given {entity_type}` or `Protocols from protocols.io uploaded for the given {entity_type} or another wording work better?